### DISCUSSION TITLE
Instructions and examples of settings

### DISCUSSION THREAD

**Discussion by cjwworld** [shiningw/ncdownloader#69] [Q&A]
Is there a tutorial on how to use this?  I am using it for Youtube download, I inputed one URL and then it downloaded over 300 videos.  Is there a way to limit how many downloads?  and is there a way to delete multiple findings?  I got 12 pages worth to go thru.  I can't find any information.  
Thanks

**Comment by Nocturna22** [comment]
Could it be that you accidentally entered a playlist in the text field and then the playlist was accidentally downloaded?

Try copying the video link out of the URL.
